internal/captain/command.go

```diff
@@ -711,6 +711,12 @@ func setupSensibleErrors(err error) error {
 		)
 	}
 
+	if strings.Contains(errMsg, "flag needs an argument: ") {
+		flag := strings.SplitN(errMsg, ": ", 2)[1]
+		return locale.NewInputError(
+			locale.Tl("command_flag_needs_argument", "Flag needs an argument: [NOTICE]{{.V0}}[/RESET]", flag))
+	}
+
```

Comment:
Please include a comment with an example of the string being checked for as seen for another similar case in lines 677-678.

Reply:
Also, please defensively handle indexing of the result of SplitN.

Reply:
I am under the impression that it is impossible for the result of SplitN to panic because of the `string.Contains()` call above it. When you mean defensively, do you mean in case the error message changes over time?

Reply:
I see; The worst case here would be an empty string. Feel free to check for empty string or not.